reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.md

```diff
@@ -25,23 +25,23 @@ docs: https://cloud.ibm.com/docs/sap-powervs
 image_source: https://github.com/terraform-ibm-modules/terraform-ibm-powervs-sap/blob/main/reference-architectures/sap-ready-to-go/deploy-arch-ibm-pvs-sap-ready-to-go.svg
 
 related_links:
```

Comment:
use the original capitalization for this whole section. Especially the property names.